pkg/handlers/generic/mutation/controlplanevirtualip/inject_test.go

```diff
@@ -100,7 +100,7 @@ var _ = Describe("Generate ControlPlane virtual IP patches", func() {
 			cluster: &clusterv1.Cluster{
 				ObjectMeta: metav1.ObjectMeta{
 					Name:      request.ClusterName,
-					Namespace: metav1.NamespaceDefault,
+					Namespace: request.Namespace,
```

Comment:
Why wasn't this causing the test to fail before?

Reply:
The values coincided

Reply:
Aha! Thanks for explaining!